ansible/main.yml

```
@@ -0,0 +1,110 @@
+---
+- hosts: all
+  vars:
+    app_path: /home/ubuntu/RD-WATCH
+    venv: "{{ app_path }}/venv"
+  tasks:
+  - name: checkout code
+    git:
+      repo: "https://github.com/ResonantGeoData/RD-WATCH.git"
+      dest: "{{ app_path }}"
+      force: yes
+
+  - name: Set environment variables
+    set_fact:
+      watch_environment:
+        DJANGO_CONFIGURATION:  "{{ lookup('env', 'DJANGO_CONFIGURATION') }}"
+        DJANGO_DATABASE_URL: "{{ lookup('env', 'DJANGO_DATABASE_URL') }}"
+        DJANGO_CELERY_BROKER_URL: "{{ lookup('env', 'DJANGO_CELERY_BROKER_URL') }}"
+        DJANGO_MINIO_STORAGE_ENDPOINT: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ENDPOINT') }}"
+        DJANGO_MINIO_STORAGE_ACCESS_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_ACCESS_KEY') }}"
+        DJANGO_MINIO_STORAGE_SECRET_KEY: "{{ lookup('env', 'DJANGO_MINIO_STORAGE_SECRET_KEY') }}"
+        DJANGO_STORAGE_BUCKET_NAME: "{{ lookup('env', 'DJANGO_STORAGE_BUCKET_NAME') }}"
+
+  - name: install pip + python, postgres headers 
+    apt: 
+      pkg:
+        - python3-pip
+        - python3-dev
```

```suggestion
        - python3-pip
        - python3-dev
        - python3-setuptools
        - python3-venv
```
This will get the virtualenv packages for us.